internal/config/node/sysctl.go

```diff
@@ -0,0 +1,37 @@
+package node
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+var checkFsMayDetachMountsErr error
+
+func checkFsMayDetachMounts() bool {
+	// this sysctl is specific to RHEL7 kernel
+	const file = "/proc/sys/fs/may_detach_mounts"
+	data, err := ioutil.ReadFile(file)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			logrus.WithError(err).Debug("checkFsMayDetachMounts")
+		}
+		return false
```

Comment:
Do we want the check to fail if the file isn't present?

Reply:
We do not. This parameter is only available for RHEL7 kernel, so, if absent, it means we're not running on RHEL7 (or this is very old RHEL7 kernel, or we are inside some kind of container/chroot).

Reply:
The return value is not used anywhere (AFAICS). We only fail if we set the `checkFsMayDetachMountsErr` error, and it is only set if the value is read and parsed but is not 1.

Reply:
Added a comment and changed a couple of falses to trues.